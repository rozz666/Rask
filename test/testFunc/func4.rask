abc() -> void { }
abc() -> void { }
main() -> void { }

