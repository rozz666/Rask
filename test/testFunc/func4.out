func4.rask:2:1: error: redefinition of abc()
func4.rask:1:1: error: abc() previously defined here

